src/lxd.py

```diff
@@ -531,6 +532,32 @@ def delete(self):
         self._pylxd_storage_pool.delete()
 
 
+class LxdImageManager:  # pylint: disable=too-few-public-methods
+    """LXD image manager."""
+
+    def __init__(self, pylxd_client: pylxd.Client):
+        """Instantiate the LXD image manager.
+
+        Args:
+            pylxd_client: Instance of pylxd.Client.
+        """
+        self._pylxd_client = pylxd_client
+
+    def create(self, name: str, path: Path):
+        """Import a LXD image.
+
+        Args:
+            name: Alias for the image.
+            path: Path of the LXD image file.
+
+        Raises:
+            LxdError: Unable to import the file as LXD image.
+        """
+        result = secure_run_subprocess(["lxc", "image", "import", "--alias", name, str(path)])
+        if result.returncode != 0:
+            raise LxdError(result.stdout.decode("utf-8"))
```

Comment:
Hmm, perhaps we don't need a class for this? It seems that it is really just the 3 lines of logic:

```python
 result = secure_run_subprocess(["lxc", "image", "import", "--alias", name, str(path)])
 if result.returncode != 0:
    raise LxdError(result.stdout.decode("utf-8"))
```

What was the reasoning for creating it please?

Reply:
I am trying to keep all interaction with LXD in a interface file to better keep track of it.
This will make it easier to add new integration to OpenStack or other virtualisation solutions in the future.